### PR TITLE
Add support for the binary files

### PR DESCRIPTION
**Pull request by rshym**
This patch adds the possibility to flash binary files using the `xls3` command of flash_writer.

README.md is updated accordingly.

### CODE REVIEW

**Comment by rshym** [comment]
Tested with `rcar_flash --conf rcar_flash_bin.yaml ....`. Works as expected.

File: README.md [hunk 1]
```diff
@@ -340,3 +340,18 @@ For each board there are multiple options possible:
...
       different loaders.
+
+### Work with binary files
+
+In case of need, you can flash not only .srec-files but also binaries.
+For this you should use a slightly modified `flash_target`:
+
+- replace `xls2` with `xls3`
+
+- replace `img_addr` with `file_size`
```
Comment (lorc) [inline README.md:351]: Would you please update `flash_target` subsection? There is paragraph "Allowed arguments for `send` are:"

**Review by lorc — COMMENTED** [review]
It was easier than I expected :)

Anyways, I am not sure if we need a separate YAML files for two cases. But I have no strong opinion, honestly.  Both approaches have drawbacks.

If you want to leave this in two separate files, please do the following:

 - rename `h3ulcb_4x2_bin` to `h3ulcb_4x2`, as we already have suffix `_bin` in the yaml name.
 - document `--conf` option in the README.md (sorry, I missed it)

If we want to use the same YAML file, we can add `desc` field to a board entry, so we can show the difference between different "boards" in `list-boards` command.

File: README.md (hunk 1 above)
Comment (rshym) [reply README.md:354]: Done.

**Comment by rshym** [comment]
As agreed during the off-site discussion, I have removed not needed file `rcar_flash_bin.yaml`, and put an example of `flash_target` into README.md

**Comment by lorc** [comment]
Thanks! 

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by rshym** [comment]
`Ack` is added. Tested. Ready to be merged.

**Pull request merged**